site2/docs/io-debezium-source.md

```diff
@@ -170,15 +170,15 @@ This example shows how to change the data of a MySQL table using the Pulsar Debe
        ```bash
        bin/pulsar-admin source localrun \
        --archive connectors/pulsar-io-debezium-mysql-@pulsar:version@.nar \
-       --name debezium-mysql-source --destination-topic-name debezium-mysql-topic \
```

Comment:
This subcommand also exists in `io-cdc-debezium.md`, which seems to be a duplicate of `io-debezium-source.md` but still referenced in `io-cdc.md`. Maybe not in this PR's scope but can be revisited.

Reply:
@momo-jun Update `io-cdc-debezium.md`. For the duplications, let's handle them later.